src/platforms/android/migration.mdx

```diff
@@ -7,8 +7,57 @@ redirect_from:
 
 ## Migrating from `io.sentry:sentry` `4.3.0` to `io.sentry:sentry` `5.0.0`
 
+If you're using the [Android Native Development Kit](/platforms/android/using-ndk/) support and you've set the `android:extractNativeLibs="true"` meta-data in the `AndroidManifest` file or `android.bundle.enableUncompressedNativeLibs=false` in the `gradle.properties` file, you don't have to anymore, you may remove them.
+Now Sentry is able to symbolicate events with the default value of [extractNativeLibs](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs) and [android.bundle.enableUncompressedNativeLibs](https://developer.android.com/studio/releases/gradle-plugin#behavior-changes)
+
 `Sentry#startTransaction` by default does not bind created transaction to the scope. To start transaction with binding to the scope, use one of the new overloaded `startTransaction` methods taking `bindToScope` parameter and set it to `true`. Bound transaction can be retrieved with `Sentry#getSpan`.
 
+`Settings.Secure.ANDROID_ID` has been removed in favor of a random generated `installationId` and it may affect the number of unique users in the the Issues page and Alerts.
```

Comment:
```suggestion
A random generated `installationId` replaces `Settings.Secure.ANDROID_ID`, which has been removed. This may affect the number of unique users displayed on the the Issues page and Alerts.
```